### ISSUE TITLE
Documentation – table layout in 'List of resource types' is broken

### ISSUE THREAD

**Issue by JdeBud** [php/web-php#688]
In the PHP manual, the table is not in its place in the "Layout Content" section of the HTML. It seems to be a simple problem with the styling of the CSS.

Take a look on [List of Resource Types](https://www.php.net/manual/de/resource.php).

![image](https://user-images.githubusercontent.com/74646666/186891603-91ec9a81-ecac-43d3-8cb7-229ccf24a6f0.png)

```html
<!DOCTYPE html>
<!--[…]-->
<div id="layout" class="clearfix">
    <section id="layout-content">
        <!--[…]-->
        <div id="resource" class="appendix">
            <!--[…]-->
            <table class="doctable table"> <!-- This is outside the frame -->
            <!--[…]-->
```

**Comment by cmb69** [comment]
Yeah, the overflowing tables are a known issue. Basically, this could be solved by applying `overflow-x:scroll`, but I'm not convinced that would be the right solution (consider mobile devices). Instead, I suggest to reconsider the use of tables; in this case, a structured list might be more appropriate (in which case that wouldn't be a web-php issue, but rather a doc-en issue, or possibly a phd issue).
